resources/repos.json

```diff
@@ -769,7 +769,7 @@
 		},
 		"dependencies": {
 			"Minecraft Version(s)": {
-				"1": "1.15.x, 1.16.x"
+				"1": "1.14+"
```

Comment:
Did you test it on MC 1.42 :LUL:
No but seriously, please stick to the current format of listing versions and only list the versions you actually know it works on.